pkg/logger/logger.go

```diff
@@ -153,11 +155,3 @@ func Error(err error) zap.Field {
 func String(k, v string) zap.Field {
 	return zap.String(k, v)
 }
-
-func Int(k string, v int) zap.Field {
```

Comment:
Considerations for how this affects internally? Or how users whom may be using this package for their custom interceptors?

Filtering with >< operators is not the same with strings vs int.

Reply:
We are pretty locked in with zap so we just zap.Int everywhere now.

Reply:
Might as well strip the others and fully adopt zap.

Reply:
Good point. Done. Thanks!